cmd/iq_test.go

```diff
@@ -35,18 +36,46 @@ import (
 )
 
 func TestIqApplicationFlagMissing(t *testing.T) {
-	output, err := executeCommand(rootCmd, "iq")
+	output, err := executeCommand(rootCmd, iqCmd.Use)
 	assert.Contains(t, output, "Error: required flag(s) \""+flagNameIqApplication+"\" not set")
 	assert.NotNil(t, err)
 	assert.Contains(t, err.Error(), "required flag(s) \""+flagNameIqApplication+"\" not set")
 }
 
 func TestIqHelp(t *testing.T) {
-	output, err := executeCommand(rootCmd, "iq", "--help")
+	output, err := executeCommand(rootCmd, iqCmd.Use, "--help")
 	assert.Contains(t, output, "go list -json -m all | nancy iq --"+flagNameIqApplication+" your_public_application_id --"+flagNameIqServerUrl+" ")
 	assert.Nil(t, err)
 }
 
+func TestIqCommandPathInvalidName(t *testing.T) {
+	origConfig := configOssi
+	defer func() {
+		configOssi = origConfig
+	}()
+	// not sure why calling executeCommand() fails as part of test suite, but is fine individually.
+	//_, err := executeCommand(rootCmd, iqCmd.Use, "--path", "invalidPath", "-a", "appId")
```

Comment:
So these are a fix it later thing??

Reply:
> So these are a fix it later thing??

Yes, or at least an "understand it better later" thing. Added TODO comments. cbcbbe3 

I'm pretty sure this is an example of the dangers of using shared state - meaning the `configOssi` is essentially global, and testing gets wonky quickly because of the side effects of manipulating `configOssi`. Combine that with some of the essentially global state of Cobra/Viper initialization, and complexity abounds.

I would prefer these tests to work using `executeCommand()` because that would verify more of the Cobra stuff, but I spent enough time trying to figure out why the `executeCommand()` flavor works when run alone, but causes failures when run in a test suite with other tests.